site/gatsby-site/src/components/BibTex.js

```diff
@@ -22,6 +22,14 @@ const BibTex = ({ nodes, incidentDate, incident_id }) => {
   // Only return the earliest submitter
   let submitterCite = getFormattedName(docs[0]['submitters'][0]);
 
+  const firstEditor = editors[0];
```

Comment:
That's generally my preference as well, but it does seem to contradict the linter. I wrapped those lines in `eslint-disable`, `eslint-enable`.

Reply:
Ah, got it. It is generally better to figure out how to make the linter work with you than to disable the linter. In this case it might be best to just be OK with the whitespace.